pkg/commands/execution_context.go

```diff
@@ -525,6 +529,15 @@ func (ctx *executionContext) completeTask(event operationEvent) {
 	ctx.Lock()
 	delete(ctx.running, event.task.taskID)
 	ctx.completed = append(ctx.completed, event.task)
+	failedTests := 0
+	for _, t := range ctx.completed {
+		if t.test.Status == model.StatusFailed {
+			failedTests++
+		}
+	}
+	if ctx.cloudTestConfig.FailedTestsLimit != 0 && failedTests >= ctx.cloudTestConfig.FailedTestsLimit {
```

Comment:
```suggestion
	if ctx.cloudTestConfig.FailedTestsLimit != 0 && failedTests == ctx.cloudTestConfig.FailedTestsLimit {
```

Reply:
Node: The operation with array `completed` is synchronized.

Reply:
Yes, you're right. Fixed it.